Scripts/Mend SCA/README.md

```diff
@@ -323,3 +325,35 @@ This script exports the product names and their last scan dates to CSV file for
 
 **Execution:** 
 python get-request-history.py MEND_URL MEND_APIKEY MEND_USERKEY DAYS_TO_QUERY
+
+<br />
+<hr />
+
+# Get all Users that are part of Organizations which are a part of a Global Organization
+
+[get-all-users-under-global-org.py](get-all-users-under-global-org.py)
+
+This script allows a user to retrieve every single user account that is a part of any of their Organizations which reside under the "umbrella" of a Global Organization. This allows an organization to take a full inventory of all users that have access to their resources.
+
+The [get-all-users-under-global-org.py](get-all-users-under-global-org.py) script can be run with Python 3.9+ and utilizes the Mend API 1.4 as well as API 2.0 to get the information necessary.
+
+> **_NOTE:_** If an error occurs with this script that produces an error when attempting to log in via API 2.0, this can happen on a rare occasion. The best thing to do is to implement a more comprehensive retry logic, or just try running the script again.
+
+<br />
+
+**Prerequisites:**  
+
+* `python 3.9+`
+
+**Execution:**
+```shell
+# Create Environment Variables
+export MEND_URL="https://saas.whitesourcesoftware.com"
+export MEND_APIKEY="<org_token>"
```

Comment:
should be WS_APIKEY as this value is not used with the CLI and is legacy from unified agent